### PR TITLE
Add premium_progress_bar_enabled attribute on Guild.

### PR DESCRIPTION
**Pull request by nerdguyahmad**
## Summary

This pull request adds the `premium_progress_bar_enabled` attribute on the `Guild` obejct.

See https://github.com/discord/discord-api-docs/pull/4001

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by bijij** [comment]
Can this be toggled by bots via a http request?

**Comment by nerdguyahmad** [comment]
> Can this be toggled by bots via a http request?

Yes it can be in `edit()`, Forgot to add that. Sorry.

File: discord/guild.py [hunk 1]
```diff
@@ -483,6 +488,7 @@ def _from_data(self, guild: GuildPayload) -> None:
         self.mfa_level: MFALevel = try_enum(MFALevel, guild.get('mfa_level', 0))
         self.approximate_presence_count: Optional[int] = guild.get('approximate_presence_count')
         self.approximate_member_count: Optional[int] = guild.get('approximate_member_count')
+        self.premium_progress_bar_enabled: bool = guild.get("premium_progress_bar_enabled", False)
```
Comment (Rapptz) [inline discord/guild.py:491]:
```suggestion
        self.premium_progress_bar_enabled: bool = guild.get('premium_progress_bar_enabled', False)
```

File: discord/guild.py [hunk 2]
```diff
@@ -1763,6 +1775,9 @@ async def edit(
 
             fields['features'] = features
 
+        if premium_progress_bar_enabled is not MISSING:
+            fields["premium_progress_bar_enabled"] = premium_progress_bar_enabled
```
Comment (Rapptz) [inline discord/guild.py:1779]:
```suggestion
            fields['premium_progress_bar_enabled'] = premium_progress_bar_enabled
```

**Review by Rapptz — CHANGES_REQUESTED** [review]
Mostly good, thanks.

**Review by Rapptz — APPROVED** [review]

**Pull request merged**